packages/pilot/src/containers/Sidebar/index.js

```diff
@@ -41,28 +41,35 @@ class SidebarContainer extends React.Component {
 
   renderSections () {
     const {
+      anticipationLimit,
       balance,
       onAnticipate,
       onWithdraw,
       t,
+      transfersPricing,
     } = this.props
 
     const getFrombalance = propOr(null, __, balance)
     const available = getFrombalance('available')
     const waitingFunds = getFrombalance('waitingFunds')
 
+    const disableBtnWithdraw = available <= transfersPricing.ted
```

Comment:
acho que teve um confusão aqui, acho que o correto era esse calculo abaix né?
da forma que esta, na inha 65 está retornando um numero e não um booleano
no caso a variavel tambem não precisa estar relacionada ao botão com `btn` no nome, acho que só `disableWithdraw` já é claro o suficiente
```suggestion
   const MINIMUM_WITHDRAWAL_VALUE = 100
    const disableWithdraw = available <= transfersPricing.ted + MINIMUM_WITHDRAWAL_VALUE
```

Reply:
consertado!